tool/license/allowed-licenses.txt

```diff
@@ -564,6 +564,36 @@
             "moduleLicense": "The Apache Software License, Version 2.0",
             "moduleVersion": "0.8.1",
             "moduleName": "io.prometheus:simpleclient_httpserver"
+        },
+        {
+            "moduleLicense": "The Apache License, Version 2.0",
+            "moduleVersion": "1.3.0",
+            "moduleName": "io.opentelemetry:opentelemetry-exporter-zipkin"
+        },
+        {
+            "moduleLicense": "The Apache Software License, Version 2.0",
+            "moduleVersion": "2.16.3",
+            "moduleName": "io.zipkin.reporter2:zipkin-reporter"
+        },
+        {
+            "moduleLicense": "The Apache Software License, Version 2.0",
+            "moduleVersion": "2.16.3",
+            "moduleName": "io.zipkin.reporter2:zipkin-sender-okhttp3"
+        },
+        {
+            "moduleLicense": "The Apache Software License, Version 2.0",
+            "moduleVersion": "2.23.2",
+            "moduleName": "io.zipkin.zipkin2:zipkin"
+        },
+        {
+            "moduleLicense": "Apache 2.0",
```

Comment:
Here not correct, `The Apache Software License, Version 2.0` or  `The Apache License, Version 2.0`

Reply:
Yes, there might some problem with license check plugin, the one license may have different name, and we need to normalize them, we may need to add a LicenseBundleNormalizer  filter.
https://github.com/jk1/Gradle-License-Report

I will test this.